apps/api/.env.local.example

```
@@ -8,4 +8,14 @@ RESEND_API_KEY=
\ No newline at end of file
+GOOGLE_CLIENT_SECRET=
+
+# GitHub
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+
+# Frontend base URL (for redirects back to frontend after OAuth)
+VITE_BASE_URL=
+
+# Optional: API port (defaults to 3333 if not set)
+PORT=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)